launcher/package.json

```diff
@@ -19,7 +19,8 @@
     "watch:css": "npx tailwindcss -i ./src/main.css -o ./public/output.css --watch",
     "lint:fix": "eslint --ext .js,.vue --ignore-path ../.gitignore --fix src",
     "stereum": "concurrently \"npm:electron:serve\" \"npm:watch:css\"",
-    "format": "prettier .  --write"
+    "format": "prettier .  --write",
+    "electron:dev": "nodemon --exec 'npm run electron:serve' --watch src/backend --ext js"
```

Comment:
what exactly is this doing?

Reply:
@MaxTheGeeek pls answer

Reply:
we can leave it in there but i would move it to the devDependencies

Reply:
it's a nodemon.
that makes life easier for the node.js devs.

Reply:
no need to run over and over again during test : 👍

Reply:
removed for now, we can add this later